src/test/java/com/powsybl/caseserver/CaseControllerTest.java

```diff
@@ -401,23 +401,17 @@ private UUID importCase(String testCase, Boolean withExpiration) throws Exceptio
 
     @Test
     public void validateCaseNameTest() {
-        CaseService.validateCaseName("test");
         CaseService.validateCaseName("test.xiidm");
-        CaseService.validateCaseName("te-st");
         CaseService.validateCaseName("test-case.7zip");
         CaseService.validateCaseName("testcase1.7zip");
         CaseService.validateCaseName("testcase1.xiidm.gz");
+        CaseService.validateCaseName("test..xiidm");
 
```

Comment:
You can add :  
try {
            CaseService.validateCaseName("test");
            fail();
        } catch (CaseException ignored) {
        }

Reply:
done